internal/db/migrations/postgres/0/100_server_tags_migrations.up.sql

```diff
@@ -0,0 +1,101 @@
+begin;
+
+-- This series of expressions fixes the primary key on the server table
+alter table session
+  drop constraint session_server_id_server_type_fkey;
+alter table server
+  drop constraint server_pkey;
+alter table server
+  drop column name;
```

Comment:
Why are we dropping the `name` column?

Reply:
Because it is always exactly the same as the private_id column and there are no plans to change that.